br/pkg/pdutil/pd.go

```
@@ -344,7 +347,33 @@ func (p *PdController) ResumeSchedulers(ctx context.Context, schedulers []string
 	return errors.Trace(p.resumeSchedulerWith(ctx, schedulers))
 }
 
+func (p *PdController) ResumeRegionLabelRule(ctx context.Context, ruleID string) error {
```

it seems we suppress all the errors in this method, the error will always be nil, is that expected? If so we can just remove the return type

---

You are right, there used to be errors, but now all return would be nil